src/python/src/grpc/_adapter/_c_test.py

```diff
@@ -1,219 +1,40 @@
-    thread.start()
-    time.sleep(1)
-    server.stop()
-    completion_queue.stop()
-    for _ in range(_IDEMPOTENCE_DEMONSTRATION):
-      event = completion_queue.get(time.time() + _TIMEOUT)
-      self.assertIs(event.kind, _datatypes.Event.Kind.STOP)
-    thread.join()
-    del server
-    del completion_queue
-
-    _c.shut_down()
-
-  def test_client_credentials(self):
-    root_certificates = b'Trust starts here. Really.'
-    private_key = b'This is a really bad private key, yo.'
-    certificate_chain = b'Trust me! Do I not look trustworty?'
-
-    _c.init()
-
-    client_credentials = _c.ClientCredentials(
-        None, None, None)
-    self.assertIsNotNone(client_credentials)
-    client_credentials = _c.ClientCredentials(
-        root_certificates, None, None)
-    self.assertIsNotNone(client_credentials)
-    client_credentials = _c.ClientCredentials(
-        None, private_key, certificate_chain)
-    self.assertIsNotNone(client_credentials)
-    client_credentials = _c.ClientCredentials(
-        root_certificates, private_key, certificate_chain)
-    self.assertIsNotNone(client_credentials)
-    del client_credentials
-
-    _c.shut_down()
+from grpc._adapter import _types
 
-  def test_server_credentials(self):
-    root_certificates = b'Trust starts here. Really.'
-    first_private_key = b'This is a really bad private key, yo.'
-    first_certificate_chain = b'Trust me! Do I not look trustworty?'
-    second_private_key = b'This is another bad private key, yo.'
-    second_certificate_chain = b'Look into my eyes; you can totes trust me.'
 
-    _c.init()
+class CTypeSmokeTest(unittest.TestCase):
 
-    server_credentials = _c.ServerCredentials(
-        None, ((first_private_key, first_certificate_chain),))
-    del server_credentials
-    server_credentials = _c.ServerCredentials(
-        root_certificates, ((first_private_key, first_certificate_chain),))
-    del server_credentials
-    server_credentials = _c.ServerCredentials(
-        root_certificates,
-        ((first_private_key, first_certificate_chain),
-         (second_private_key, second_certificate_chain),))
-    del server_credentials
-    with self.assertRaises(TypeError):
-      _c.ServerCredentials(
-          root_certificates, first_private_key, second_certificate_chain)
+  def testClientCredentialsUpDown(self):
+    creds = _c.ClientCredentials.fake_transport_security()
+    del creds
 
-    _c.shut_down()
+  def testServerCredentialsUpDown(self):
+    creds = _c.ServerCredentials.fake_transport_security()
+    del creds
 
-  @unittest.skip('TODO(nathaniel): find and use real-enough test credentials')
-  def test_secure_server(self):
-    _c.init()
+  def testCompletionQueueUpDown(self):
+    cq = _c.CompletionQueue()
+    del cq
 
-    server_credentials = _c.ServerCredentials(
-        'root certificate', (('private key', 'certificate chain'),))
+  def testServerUpDown(self):
+    cq = _c.CompletionQueue()
```

Comment:
Our style guide disallows single-letter identifier names, and initialisms, acronyms, and abbreviations aren't any better. Spell out "completion_queue", "credentials", "server", "channel", and so forth.